### PR TITLE
Fix check for PL SysMon

### PR DESCRIPTION
**Pull request by vshpatel**

### CODE REVIEW

**Comment by msreeram-xilinx** [comment]
LICENSE CHECKS: 
Modified Files:
src/xlnx_platformstats.c: No License Change
License Check is Passed

File: src/xlnx_platformstats.c [hunk 1]
```diff
@@ -1135,11 +1135,11 @@ int print_sysmon_power_info(int verbose_flag, int sample_interval, int sample_wi
 		fprintf(fp_out, "VTT PS GTR voltage    					:     %ld mV\t\t %ld mV\n\n",VTT_PS_GTR,VTT_PS_GTR_avg);
 
 		fprintf(fp_out, "PL Sysmon\n");
-		if (PL_TEMP_avg > 0) {
-			fprintf(fp_out, "PL temperature    					:     %ld C\t\t %ld C\n\n",(PL_TEMP)/1000,(PL_TEMP_avg)/1000);
+		if (PL_TEMP_avg == -280226) {
```
Comment (chkohn) [inline src/xlnx_platformstats.c]: prob better to use a macro here and a comment that explains what this magic value is

**Review by chkohn — CHANGES_REQUESTED** [review]
see inline.

@pallav-xlnx  can you please review and merge?

**Comment by msreeram-xilinx** [comment]
LICENSE CHECKS: 
Modified Files:
src/xlnx_platformstats.c: No License Change
License Check is Passed

**Review by pallav-xlnx — APPROVED** [review]

**Pull request merged**